src/main/java/org/cryptomator/sanitizer/restorer/PathEncryptor.java

```diff
@@ -45,4 +61,25 @@ private static Path resolvePath(Path vaultRoot, Console console, Cryptor cryptor
 		}
 	}
 
+    private static void printResolvedPaths(Console console, Map<String, Path> paths, String outputPath) throws IOException {
+		if (outputPath != null) {
+			BufferedWriter writer = new BufferedWriter(new FileWriter(outputPath));
```

Comment:
Use try-with-resource and remove the corresponding `close()`

```suggestion
			try (BufferedWriter writer = new BufferedWriter(new FileWriter(outputPath))) {
```